ext/opentelemetry-ext-jaeger/src/opentelemetry/ext/jaeger/__init__.py

```diff
@@ -197,6 +197,8 @@ def _translate_to_jaeger(spans: Span):
         parent_id = span.parent.span_id if span.parent else 0
 
         tags = _extract_tags(span.attributes)
+        if span.resource is not None:
```

Comment:
I'm wondering how we can make this test more efficient. It looks to me that `resource` is never `None` but it is `_EMPTY_RESOURCE`.

https://github.com/open-telemetry/opentelemetry-python/blob/090b6640495bcb994a9467ab23e2b1abe7fc4af9/opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py#L695

https://github.com/open-telemetry/opentelemetry-python/blob/090b6640495bcb994a9467ab23e2b1abe7fc4af9/opentelemetry-sdk/src/opentelemetry/sdk/resources/__init__.py#L32-L33

I think one solution could be to expose `_EMPTY_RESOURCE` (remove the `_`) and make a comparison using the `is` operator.

`if span.resource is not ̣̣̣resources.EMPTY_RESOURCE:`

Reply:
I made the suggested change.
Though notice, I had to create an empty resource in all of the spans used for the jaeger tests, as it is not created by default when accessing the `trace.Span` directly.

I guess that in the real flow it will always be created, but maybe it's something to consider and we should keep the `is not None` check nonetheless, just to play it safe.
What do you think?

Reply:
Good point. TBH I'm not 100% what's the best approach here, I made that suggestion to make it more efficient, but probably it's not worthy as this check is very fast in both cases and we haven't done an optimization in other places yet. I'm fine with both solutions, let's see other people's opinions on this.